Proteomics/Modifications/ModificationGeneral.cs

```diff
@@ -14,9 +14,22 @@ public class ModificationGeneral
         public string ModificationType { get; private set; }
         public string FeatureType { get; private set; }
         public ModificationMotif Target { get; private set; }
-        public ModLocationOnPeptideOrProtein? Position { get; private set; }
+        public string Position { get; private set; }
```

Comment:
I changed it because it's not always on a terminus. the anywhere means anywhere. I was looking for a term that was more encompassing. I also don't like position. can you think of another option?

Reply:
yeah I agree.. maybe like LocationRestriction?